hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java

```diff
@@ -3124,7 +3124,12 @@ private RangerHiveAccessRequest buildRequestForAlterTableSetOwnerFromCommandStri
 
 	static void setOwnerUser(RangerHiveResource resource, HivePrivilegeObject hiveObj, IMetaStoreClient metaStoreClient) {
 		if (hiveObj != null) {
-			// resource.setOwnerUser(hiveObj.getOwnerName());
+			LOG.info("gbj setOwner()");
```

Comment:
Do we need these logs?

Reply:
No, but they make debugging easier.

Reply:
Once, I finish I can clean them up.

Reply:
Let's do that. I'll approve it so you can go ahead and merge when you're done.